source/hook.cpp

```diff
@@ -91,11 +91,11 @@ static uint32_t jenkings_one_at_a_time(const uint8_t* key, size_t sz) {
 }
 
 struct HotKey {
-	std::vector<key_t> keys;
+	std::vector<std::pair<key_t, bool>> keys;
 	std::unique_ptr<Nan::Callback> cbDown, cbUp;
 	bool wasDown = false;
 
-	static uint32_t Stringify(std::vector<key_t> keys) {
+	static uint32_t Stringify(std::vector<std::pair<key_t, bool>> keys) {
 		return jenkings_one_at_a_time(reinterpret_cast<uint8_t*>(keys.data()), keys.size() * sizeof(key_t));
```

Comment:
`sizeof(key_t)` needs to be `(sizeof(key_t) + sizeof(bool)`, otherwise only half of the actual hotkey will be hashed.

Reply:
Missed that, good catch.